homeassistant/components/iron_os/strings.json

```diff
@@ -79,6 +83,81 @@
         "name": "Long-press temperature step"
       }
     },
+    "select": {
+      "min_dc_voltage_cells": {
+        "name": "Battery cell count",
+        "state": {
+          "no_battery": "External Power Supply (DC)",
+          "battery_3s": "3S (3 cells)",
+          "battery_4s": "4S (4 cells)",
+          "battery_5s": "5S (5 cells)",
+          "battery_6s": "6S (6 cells)"
+        }
+      },
+      "orientation_mode": {
+        "name": "Display orientation mode",
+        "state": {
+          "right_handed": "Right-handed",
+          "left_handed": "Left-handed",
+          "auto": "Auto"
+        }
+      },
+      "animation_speed": {
+        "name": "Animation speed",
+        "state": {
+          "off": "[%key:common::state::off%]",
+          "slow": "[%key:component::iron_os::common::slow%]",
+          "medium": "Medium",
+          "fast": "[%key:component::iron_os::common::fast%]"
+        }
+      },
+      "autostart_mode": {
+        "name": "Start-up behavior",
+        "state": {
+          "disabled": "[%key:common::state::disabled%]",
+          "soldering": "Soldering mode",
+          "sleeping": "Sleeping mode",
+          "idle": "Idle mode"
+        }
+      },
+      "temp_unit": {
+        "name": "Temperature display unit",
+        "state": {
+          "celsius": "Celsius (C°)",
+          "fahrenheit": "Fahrenheit (F°)"
+        }
+      },
+      "desc_scroll_speed": {
+        "name": "Scrolling speed",
+        "state": {
+          "slow": "[%key:component::iron_os::common::slow%]",
+          "fast": "[%key:component::iron_os::common::fast%]"
+        }
+      },
+      "locking_mode": {
+        "name": "Button locking mode",
+        "state": {
+          "off": "[%key:common::state::off%]",
+          "boost_only": "Boost only",
+          "full_locking": "Full locking"
+        }
+      },
+      "logo_duration": {
+        "name": "Boot logo duration",
+        "state": {
+          "off": "[%key:common::state::off%]",
+          "seconds_1": "1 second",
+          "seconds_2": "2 second",
+          "seconds_3": "3 second",
+          "seconds_4": "4 second",
+          "seconds_5": "5 second",
+          "loop": "Loop"
+        }
+      },
+      "ui_language": {
+        "name": "Interface language"
+      }
```

Comment:
I don't know how I feel about this one yet

Reply:
has no effects on HA, just the lang on the oled display of the device.

Reply:
I know, but more about the whole list of options that need to be translated

Reply:
Can we detect it?

Reply:
Unfortunately not, the characteristic is still there, it is possible to write values to it and there won't be any errors, it will just have no effect.

Reply:
Do we need this select entity?
Selecting the language is a one-time thing in the setup, which can be done on the device (I assume you can choose it on the device). No user would automate it/change it.

In my opinion, we should only add entities especially config entries, which will be used. I know config entries are used rarely but I expect nobody will change the language via HA

Reply:
I would suggest removing it for now. We can always add it later if users need it

Reply:
By the way, one possible use case i could imagine would be using scenes to have different "preferred" settings and switch between the scenes in automations. 
But I also think it's the best to remove it for now, too much effort with all the translations.